ansible/roles/cartodb.nodejs/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+ - name: Add cartodb nodejs ppa
+   apt_repository: repo=ppa:cartodb/nodejs-010 state=present update_cache=yes
+
+ - name: Install nodejs 0.10.x
+   apt: name=nodejs state=present
```

Comment:
Is it worth pinning this to something just to prevent the wrong package from being installed? The CartoDB one will probably get installed every time as things stand now because of the version mismatch:

```bash
vagrant@cartodb:~$ sudo apt-cache show nodejs | grep Version
Version: 0.10.26-1chl1~precise1
Version: 0.6.12~dfsg1-1ubuntu1
```